modules/apps/polls/polls-web/src/main/resources/META-INF/resources/polls_display/view.jsp

```diff
@@ -80,19 +82,19 @@ if (question != null) {
 			<aui:fieldset-group markupView="lexicon">
 				<aui:fieldset>
 					<h2>
-						<%= StringUtil.replace(question.getDescription(locale), CharPool.NEW_LINE, "<br />") %>
+						<%= StringUtil.replace(questionDescription.toString(), CharPool.NEW_LINE, "<br />") %>
 					</h2>
 
 					<c:choose>
 						<c:when test="<%= !question.isExpired() && !hasVoted && PollsQuestionPermission.contains(permissionChecker, question, ActionKeys.ADD_VOTE) %>">
 
 							<%
 							for (PollsChoice choice : choices) {
-								choice = choice.toEscapedModel();
+								UserInputString choiceDescription = choice.getDescription(locale);
 							%>
 
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: